src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -865,7 +865,7 @@ class IMain(initialSettings: Settings, protected val out: JPrintWriter) extends
         |${preambleHeader format lineRep.readName}
         |${envLines mkString ("  ", ";\n  ", ";\n")}
         |$importsPreamble
-        |${indentCode(toCompute)}""".stripMargin
+        |%s""".stripMargin.format(indentCode(toCompute))
```

Comment:
Good catch! I was just in the same area in my repl-modularize branch. I would get rid of `stripMargin` entirely and assemble with `+`. The next line also looks fishy:
```
      def preambleLength = preamble.length - toCompute.length - 1
```

Where does this `-1` come from? The assumption that `indentCode` appends a `\n`?